tools/trace_analyzer_test.cc

```diff
@@ -669,6 +680,144 @@ TEST_F(TraceAnalyzerTest, DeleteRange) {
   CheckFileContent(top_qps, file_path, true);
 }
 
+// Test analyzing of Iterator
+TEST_F(TraceAnalyzerTest, Iterator) {
+  std::string trace_path = test_path_ + "/trace";
+  std::string output_path = test_path_ + "/iterator";
+  std::string file_path;
+  std::vector<std::string> paras = {"./trace_analyzer",
+                                    "-analyze_get",
+                                    "-analyze_put",
+                                    "-analyze_delete",
+                                    "-analyze_merge",
+                                    "-analyze_single_delete",
+                                    "-analyze_range_delete",
+                                    "-analyze_iterator",
+                                    "-convert_to_human_readable_trace",
+                                    "-output_key_stats",
+                                    "-output_access_count_stats",
+                                    "-output_prefix=test",
+                                    "-output_prefix_cut=1",
+                                    "-output_time_series",
+                                    "-output_value_distribution",
+                                    "-output_qps_stats",
+                                    "-no_key",
+                                    "-no_print"};
+  Status s = env_->FileExists(trace_path);
+  if (!s.ok()) {
+    GenerateTrace(trace_path);
+  }
```

Comment:
Since the initial setup steps are the same for most of the other tests, you can pull the common code into one function, and I guess you can pass in the needed params to it.